editor.planx.uk/src/routes/flowSettings.tsx

```diff
@@ -18,58 +26,66 @@ const flowSettingsRoutes = compose(
 
   mount({
     "/": redirect("./service"),
-    "/:tab": route(async (req) => {
-      const { data } = await publicClient.query({
-        query: gql`
-          query GetFlow($slug: String!, $team_slug: String!) {
-            flows(
-              limit: 1
-              where: {
-                slug: { _eq: $slug }
-                team: { slug: { _eq: $team_slug } }
+    "/:tab": map(async (req) => {
+      const isAuthorised = useStore.getState().canUserEditTeam(req.params.team);
+      if (!isAuthorised)
+        throw new NotFoundError(
+          `User does not have access to ${req.originalUrl}`,
+        );
```

Comment:
I noticed that this was missing and picked up the fix here. On staging and prod you can navigate to `<flow>/settings` without the proper permissions.